File management system/Student File Management System/Student File Management System/Program.cs

```diff
@@ -0,0 +1,179 @@
+﻿
+using System.Text.Json;
+using System.Xml.Serialization;
+using System.IO;
+using System.Xml;
+using System.Xml.Serialization;
+using System.Data.SqlTypes;
+using System.Diagnostics;
+namespace Student_File_Management_System
+{
+    class Program
+    {
+
+        static void Main(string[] args)
+        {
+            // Display the number of command line arguments.
+            bool loop = true;
+            string path = "C:\\Users\\nijme\\Desktop\\OpdrachtenStage\\File management system\\Student File Management System\\dataFiles\\";
+            Student k = null;
```

Comment:
Student k is bad naming convention, please change it to a suitable naming convention

Reply:
Fixed